module/surveys/helper.js

```diff
@@ -1550,6 +1552,10 @@ module.exports = class SurveysHelper {
           } else {
             // let solutionData = solutionDocument[0];
             const solutionsHelper = require(MODULES_BASE_PATH + '/solutions/helper');
+            if(solutionDocument[0].isAPrivateProgram){
```

Comment:
@borkarsaish65 Is this discussed with @aks30 he asked you to use isAPrivateProgram right, then do we need to update referenceFrom?